tests/functional/heketi/test_heketi_brick_evict.py

```diff
@@ -110,10 +110,29 @@ def _wait_for_gluster_pod_after_node_reboot(self, node_hostname):
             openshift_ops.check_service_status_on_pod(
                 self.ocp_client, gluster_pod, service, "active", state)
 
+    def _bring_down_node_and_wait_to_become_notready(self, hostname):
+        # Bring down the glusterfs node
+        vm_name = node_ops.find_vm_name_by_ip_or_hostname(hostname)
+        self.addCleanup(
+            self._wait_for_gluster_pod_after_node_reboot, hostname)
+        self.addCleanup(node_ops.power_on_vm_by_name, vm_name)
+        node_ops.power_off_vm_by_name(vm_name)
+
+        # Wait glusterfs node to become NotReady
+        custom = r'":.status.conditions[?(@.type==\"Ready\")]".status'
+        for w in waiter.Waiter(300, 20):
+            status = openshift_ops.oc_get_custom_resource(
+                self.ocp_client, 'node', custom, hostname)
+            if status[0] in ['False', 'Unknown']:
+                break
+        if w.expired:
+            raise exceptions.ExecutionError(
+                "Failed to bring down node {}".format(hostname))
+
     @pytest.mark.tier4
-    def test_brick_evict_with_node_down(self):
+    def test_brick_evict_with_node_down_if_three_node(self):
```

Comment:
Change TC name to - `test_brick_evict_with_node_down_with_three_node`

Reply:
Done